continuousflex/protocols/protocol_pdb_dimred.py

```diff
@@ -64,7 +51,7 @@ def _defineParams(self, form):
         form.addSection(label='Input')
         form.addParam('pdbSource', EnumParam, default=0,
                       label='Source of PDBs',
-                      choices=['Used for subtomogram synthesis', 'File pattern'],
+                      choices=['Used for subtomogram synthesis', 'File pattern', 'Object', 'Trajectory Files'],
                       help='Use the file pattern as file location with /*.pdb')
         form.addParam('pdbs', params.PointerParam, pointerClass='FlexProtSynthesizeSubtomo',
```

Comment:
I think we can add another pointerClass and let this work for synthesized images not only subtomograms right?

Reply:
Yes it would be good, we can put that as an issue for later as I won't have time to do it right away

Reply:
Lets first try to do that:

Reply:
Okay, I understand now what you said. Actually, wouldn't it be even better to output a set of pdbs from these synthesize protocols?
What I understood at the beginning was to allow synthesizing from the same input that pdb dimred (pdb object, dcd trajectory, etc.) That would be also helpful (e.g. generate a MD simulation, then synthesize images/subtomograms)

Reply:
It is a good idea to output the set of PDBs from the synthesize protocols. An exception can be made if the user does not have a lot of space, and we can finally delete them to save disk space.